tools/homer/environment.yml

```diff
@@ -0,0 +1,13 @@
+# You can use this file to create a conda environment for the homer module:
+#   conda env create -f environment.yml
+name: nfcore-module-homer
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  ## basic packages
+  - pigz=2.3.4
```

Comment:
do you need this?

Reply:
probably not - I just copied from your bedtools module

Reply:
removed and pushed